src/components/HomePage/WelcomeSection.vue

```diff
@@ -35,7 +36,8 @@ export default {
     WelcomeImage,
   },
   setup() {
-    return {};
+    const { t } = useI18n();
+    return { t };
```

Comment:
I think you have the access to the `$t` anyway and we don't need to expose it via setup or other ways.
The `useI18n` is for cases when you need to translate or do something in your `<script>`

Reply:
ooh, that makes things easier, thanks for the tip!